README.md

```diff
@@ -45,21 +45,22 @@ project(':VoiceModule').projectDir = new File(rootProject.projectDir, '../node_m
 ...
 dependencies {
     ...
-    compile project(':react-native-voice')
+    compile project(':VoiceModule')
```

Comment:
Is it correct?

I checked the example project and my project using this library, it shows me the old one: 

https://github.com/wenkesj/react-native-voice/blob/551cd7bad87a29cd11ca35084b98566c607ecdfa/VoiceTest/android/settings.gradle#L1-L4

Reply:
The fact is, that naming in `android/setting.gradle` and `android/app/build.gradle` should be the same. It is not in documentation currently. I did not look into example project. :)

So, for being consistent, I will update this PR and change naming few lines higher.